fix: update `view-transitions.mdx`

jsparkdev opened the pull request:
<!-- Thank you for opening a PR! We really appreciate you taking the time to help out 🙌 -->

#### Description (required)

- Fixed wrong links and added missing links
- Fixed invalid whitespace, typos
- Updated codeblocks for readability

astrobot-houston commented:
<!-- lunaria-action-comment -->

## Lunaria Status Overview


🌕 **This pull request will trigger status changes.**

<details>
<summary>Learn more</summary>

By default, every PR changing files present in the [Lunaria configuration's `files` property](https://lunaria.dev/reference/configuration/#files-required) will be considered and trigger status changes accordingly.

You can change this by adding one of the keywords present in the [`ignoreKeywords` property in your Lunaria configuration file](https://lunaria.dev/reference/configuration/#ignorekeywords) in the PR's title (ignoring all files) or by [including a tracker directive](https://lunaria.dev/guides/tracking/#tracker-directives) in the merged commit's description. 
</details>

### Tracked Files




| File                                                                                                                                                                        | Note                                                       |
| --------------------------------------------------------------------------------------------------------------------------------------------------------------------------- | ---------------------------------------------------------- |
| [en/guides/view-transitions.mdx](https://github.com/withastro/docs/blob/019a67852eb238b438e947ad5fa8cce318893586/src%2Fcontent%2Fdocs%2Fen%2Fguides%2Fview-transitions.mdx) | Source changed, localizations will be marked as outdated.  |

<details>
<summary>Warnings reference</summary>

| Icon | Description                                                                                                                                       |
| ---- | ------------------------------------------------------------------------------------------------------------------------------------------------- |
| 🔄️  | The source for this localization has been updated since the creation of this pull request, make sure all changes in the source have been applied. |

</details>

ArmandPhilippot commented on src/content/docs/en/guides/view-transitions.mdx:
````diff
@@ -125,20 +125,33 @@ Not all state can be preserved in this way. The restart of CSS animations and th
 You can also [manually identify corresponding elements](#naming-a-transition) if the island/element is in a different component between the two pages.
 
 ```astro title="src/pages/old-page.astro" "Video" 'transition:name="media-player"'
````
We also need to fix the highlighting:
````suggestion
```astro title="src/pages/old-page.astro" "video" 'transition:name="media-player"'
````

ArmandPhilippot commented on src/content/docs/en/guides/view-transitions.mdx:
```diff
@@ -429,41 +440,41 @@ The `<ClientRouter />` router will trigger in-page transitions from `<form>` ele
+  action="/contact"
+  method="POST"
+  enctype="application/x-www-form-urlencoded"
+>
```
I'm not sure we should remove the closing tag here:
```suggestion
>
  <!-- -->
</form>
```
I think usually we show valid syntax so the tag should be closed even if this not the main purpose of the example.

ArmandPhilippot commented on src/content/docs/en/guides/view-transitions.mdx:
````diff
@@ -429,41 +440,41 @@ The `<ClientRouter />` router will trigger in-page transitions from `<form>` ele
 You can opt out of router transitions on any individual form using the `data-astro-reload` attribute:
 
 ```astro title="src/components/Form.astro"
 <form action="/contact" data-astro-reload>
````
Same here, I think we should keep the closing tag:
```suggestion
<form action="/contact" data-astro-reload>
  <!-- -->
</form>
```

ArmandPhilippot reviewed (commented):
I left a few nits, otherwise looks good to me!

Although I'm not sure what happened with the `<Steps />` component in the `Client-side navigation process` section. In the live version of the docs, each step is wrapped in a `<p>` tag, making them more spaced out (like on other pages, e.g. https://docs.astro.build/en/tutorial/1-setup/4/#create-a-repository-on-github). However, in the preview, there is no `<p>` tag, so the steps look a bit squashed. See:
* https://deploy-preview-11231--astro-docs-2.netlify.app/en/guides/view-transitions/#client-side-navigation-process
* https://docs.astro.build/en/guides/view-transitions/#client-side-navigation-process

jsparkdev commented:
@ArmandPhilippot 

Thanks for the nice comment.

As you suggested, each step in `<Steps />` would look better if we reinstated it and added spacing between them. :+1: 

I removed the closing tag for `<form>` because I didn't think it was necessary in a code block that describes the usage of an element, and I use the same method in other code blocks. (See image below)

![a](https://github.com/user-attachments/assets/82125366-5bf4-405c-a07b-a4a60874a7cf)
![bb](https://github.com/user-attachments/assets/69a044af-115e-4bd9-a3ce-0d399bf35afd)
![cc](https://github.com/user-attachments/assets/1528279d-3763-4f70-afdb-ac2c6f911199)
![dd](https://github.com/user-attachments/assets/faa21271-b1f8-4b54-9f7c-e7ce38b1385e)


If I were to add the closing tag back in, I would also update the elements shown in the above image.

That's how I see it, but I'm sure there's a better option, what do you think?

ArmandPhilippot commented:
@jsparkdev Oh you're right, I didn't pay attention to that... So, yeah, I'm fine with that. 😄 I marked these two suggestions as resolved!

ArmandPhilippot reviewed (approved):
LGTM, thanks!
